.ci/wercker.sh

```diff
@@ -100,11 +100,11 @@ no-error-apex-core)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/apache/incubator-apex-core.git
-  cd .ci-temp/incubator-apex-core
+  checkout_from https://github.com/apache/apex-core
```

Comment:
@nmancus1 I believe checkstyle forks are all part of the checkstyle organization. Ex: https://github.com/checkstyle/apex-core

Reply:
Yes, that's right.  I had the old version forked as well, and it caused me some confusion. Fixed.